src/components/CategoryComponentContainer/Component.ts

```diff
@@ -0,0 +1,108 @@
+                        fixedSize: true,
+                        inputFormat: 'plain',
+                        validate: {
+                            minWords: '',
+                            maxWords: '',
+                        },
+                        logic: [
+                            {
+                                name: 'componentChange event',
+                                trigger: {
+                                    type: 'event',
+                                    event: 'componentChange',
+                                },
+                                actions: [
+                                    {
+                                        name: 'update value components',
+                                        type: 'customAction',
+                                        customAction:
+                                            `const change = result[0]\nif(change && change.component.key === '${CHECKBOXES_VC}')` +
+                                            ' {\n  const vcs = change.component.values.filter(vc => change.value[vc.value] && vc.value !== "other")\n' +
+                                            '  instance.valueComponents = [...vcs]\n  instance.redraw()\n}',
+                                    },
+                                ],
+                            },
+                        ],
+                    },
+                    {
+                        label: 'Tell us more about the other concerns that you have.',
+                        key: 'textareavc1',
+                        customConditional: `show = data && data['${CHECKBOXES_VC}'] && data['${CHECKBOXES_VC}']['other']`,
+                        type: 'textareavc',
+                        rows: 3,
+                        wysiwyg: false,
+                        editor: '',
+                        spellcheck: true,
+                        fixedSize: true,
+                        inputFormat: 'plain',
+                        validate: {
+                            minWords: '',
+                            maxWords: '',
+                        },
+                        logic: [
+                            {
+                                name: 'componentChange event',
+                                trigger: {
+                                    type: 'event',
+                                    event: 'componentChange',
+                                },
+                                actions: [
+                                    {
+                                        name: 'update value components',
+                                        type: 'customAction',
+                                        customAction:
+                                            `const change = result[0]\nif(change && change.component.key === '${CHECKBOXES_VC}')` +
+                                            ' {\n  const vcs = change.component.values.filter(vc => vc.value === "other")\n' +
```

Comment:
value 'other' could be a constant since its used in a few places

Reply:
Absolutely, changed